### ISSUE TITLE
Picker and Select issues

### ISSUE THREAD

**Issue by adamkearsley** [FokkeZB/nl.fokkezb.form#11]
<h1>PICKER</h1>
When adding a picker (as per test app)
```
{
        name: 'date',
        label: 'Picker type (date)',
        type: 'picker',
        picker: {
          type: Ti.UI.PICKER_TYPE_DATE,
          valueFormat: 'DD-MM-YYYY'
        },
        value: '04-07-2014'
}
```
i get error
```
[ERROR] :  Script Error {
[ERROR] :      column = 24;
[ERROR] :      line = 24;
[ERROR] :      message = "undefined is not an object (evaluating '$.pickerRow.add')";
[ERROR] :      stack = "focus\nonTableSingletap";
[ERROR] :  }
```
the $.pickerRow object doesn't appear to be reference anywhere?
Its not in the view picker.xml view and one isn't passed to it.

<h1>SELECT</h1>
When adding a select (as per test app)
```
{
      name: 'cities',
      label: 'Select type (obj)',
      type: 'select',
      options: {
        'ams': 'Amsterdam',
        'nyc': 'New York City',
        'lnd': 'London'
      },
      value: 'nyc'
}
```
i get no errors, a select is shown with a title and a cancel but no options?

**Comment by adamkearsley** [comment]
Created a PR
https://github.com/FokkeZB/nl.fokkezb.form/pull/12

**Comment by FokkeZB** [comment]
Fixed by #12
